deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
got moved in https://github.com/JuliaLang/julia/commit/4282ba2c47e7077203fde476a528e34c039280dd apparently